src/components/maincontroller.ts

```diff
@@ -831,7 +834,9 @@ export function createMediaInformation(
         );
     }
 
-    mediaInfo.customData.startPositionTicks = streamInfo.startPosition || 0;
+    // If the client actually sets startPosition:
+    // if(streamInfo.startPosition)
+    //     mediaInfo.customData.startPositionTicks = streamInfo.startPosition
```

Comment:
Dead code shouldn't exist in a repository in my opinion, but it you'd rather leave it that's fine.

Reply:
I think streamInfo.startPosition was an unknown variable to me, and I think the client would have passed this position, but it's a duplicate property imo. I assumed maybe someone knew something more about it, but if it actually doesn't do anything then it should be removed. :)

Reply:
If nobody has any relation to streamInfo.startPosition, and nothing is found in jellyfin-web, I think it should be deleted. :)